apps/mull-ui/src/app/pages/register/register.scss

```
@@ -0,0 +1,67 @@
+@import 'variables';
+
+.register-mull-logo {
+  padding-top: 5rem;
+  display: block;
+  margin: 0 auto;
+  width: 30%;
+  padding-bottom: 3rem;
+}
+
+.register-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-row-gap: 1.5rem;
+  margin: 0 auto;
+  width: 95%;
```

```suggestion
  display: grid;
  grid-template-columns: 1fr;
  grid-row-gap: 1.5rem;
  padding-top: 2.5rem;
  padding-left: 2rem;
  padding-right: 2rem;
```

You applied my suggestion in the login page, but not here in the register page

---

Oops my bad